doc/source/whatsnew/v0.17.0.txt

```diff
@@ -218,7 +218,9 @@ Other enhancements
 
 - Support pickling of ``Period`` objects (:issue:`10439`)
 
-- ``DataFrame.apply`` will return a Series of dicts if the passed function returns a dict and ``reduce=True`` (:issue:`8735`).
+- Added ability to automatically create a table using the gbq.to_gbq function if destination table does not exist (:issue:`8325`).
+- Added ability to replace an existing table and schema when calling the gbq.to_gbq function via the if_exists parameter (:issue:`8325`).
+- Added the following functions to the gbq module: ``gbq.table_exists()``, ``gbq.create_table()``  and ``gbq.delete_table()`` (:issue:`8325`).
```

Comment:
add a mention in the breaking changes section that the exceptions are now ``ValueError`` rather than ``IOError``